e2e_test/test_smoketest.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eo pipefail
+
+source shared.sh
+prepare_node
+
+(cd debug-fee-currency && forge build .)
```

Comment:
This might solve the CI error:
```suggestion
(cd debug-fee-currency && forge build)
```

Alternatively, upgrading to the same foundry version we use in optimism now (143abd6a768eeb52a5785240b763d72a56987b4a) could help, since that version works for me locally even with out changes.